README.md

```diff
@@ -28,9 +28,9 @@ It uses [Immer](https://immerjs.github.io/immer/) as an interface for state muta
 ```js
 import store, { sync } from "immerhin";
 
-// Create containers for each state. Sync engine only cares that the result has a "value" and a "dispatch(newValue)"
-const container1 = createContainer(initialValue);
-const container2 = createContainer(initialValue);
+// Create containers for each state. Sync engine only cares that the result has a "get()" and a "set(newValue)"
+const container1 = atom(initialValue);
```

Comment:
whats `atom` here? mb add import { atom } from '';

Reply:
Example with import below. But yeah, need to tweak docs some day.